gems/pending/VMwareWebService/VimService.rb

```diff
@@ -1215,19 +1215,20 @@ def unmarshal_response(node, vType = nil)
 
     node.children.each do |c|
       next if c.blank?
+      name = c.name.freeze
 
-      ai = aih[c.name] if aih
+      ai = aih[name] if aih
 
-      unless (v = obj[c.name])
-        v = obj[c.name] = VimArray.new("ArrayOf#{ai[:type]}") if ai && ai[:isArray]
+      unless (v = obj[name])
+        v = obj[name] = VimArray.new("ArrayOf#{ai[:type]}") if ai && ai[:isArray]
```

Comment:
[VimArray will symbolize it](https://github.com/ManageIQ/manageiq/blob/3007256ae88ffd3adc7175a3e9c6bc6832b1c6ca/gems/pending/VMwareWebService/VimTypes.rb#L64)...we could probably to_sym it right here though to save 1 alloc per loop.